tools/osbuilder/rootfs-builder/scripts/init_trusted_storage.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
```

Comment:
Is this script a copy of https://github.com/confidential-containers/guest-components/blob/main/confidential-data-hub/storage/scripts/luks-encrypt-storage ? if so then I think we should fetch the file from guest-components otherwise it can get easily out of sync and introduce regressions hard to find.

One question that I have is if any plan to replace the script by a rust app or even a function in guest-components.

Reply:
@wainersm I think we can enhance this further in our follow-up work. WDYT?

Reply:
>Is this script a copy of https://github.com/confidential-containers/guest-components/blob/main/confidential-data-hub/storage/scripts/luks-encrypt-storage ? if so then I think we should fetch the file from guest-components otherwise it can get easily out of sync and introduce regressions hard to find.

Yes, I agree with you. Although the PR includes many commits, I recommend addressing the work in the next PR. WDYT?

Reply:
Hi @ChengyuZhu6 ! I'm okay to address these comments in a follow on PR

Reply:
@ChengyuZhu6 created an issue to track that improvement: https://github.com/confidential-containers/guest-components/issues/670